### ISSUE TITLE
Add alt tags to images

### ISSUE THREAD

**Issue by rijkvanzanten** [mbergevoet/iCOV-redesign#10]
https://github.com/mbergevoet/iCOV-redesign/blob/bd954f5746ba5e0d3ea256646d86688473bb9ff9/index.html#L263

This will allow the browser to fall back to a text representation of the image in case the image fails to load, and helps visual impairment helpers like screenreaders understand and communicate the contents of the image properly
